app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/view/DisplayImageActivity.kt

```diff
@@ -0,0 +1,20 @@
+package com.github.HumanLearning2021.HumanLearningApp.view
```

Comment:
You should merge this activity with DisplayPictureActivity and adapt DisplayPictureActivityTest, this will benefit both.

Reply:
Is this still relevant, or has it been addressed ?